test/library/packages/LinearAlgebra/performance/dot-perf.execopts

```diff
@@ -1 +1 @@
---n=100000 --iters=1 -nl 4 --correctness=true
+--n=1024 --iters=0 --correctness=true
```

Comment:
Let's drop this size even lower. Note that correctness tests are compiled without fast and this is taking time in the order of minutes.